src/QuantumSdk/README.md

```diff
@@ -118,59 +118,73 @@ in the project that implements the compilation step.
 
 The Sdk defines the following properties for each project using it:
 
-- `QSharpLangVersion`:
+- `QSharpLangVersion`:    
 The version of the Q# language specification.
 
```

Comment:
(Curiosity)
I know why 2 spaces may be required in the end of the lines in some implementations of the Markdown.
But what is the purpose of more than 2 spaces?  ;-)

Reply:
I think habit, since I had a setting where I think 4 were required? :) 
Yes, the spaces are just there to have a line break.